### PR TITLE
Update to 0.12.1 tag

### PR DESCRIPTION
**Pull request by sacha-l**
Updates the install command + other references to 0.12.0 tag.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +50,15 @@ MacOS Homebrew users can use:
 
 ### Linux
 
-    wget https://github.com/chevdor/subwasm/releases/download/v0.12.0/subwasm_linux_amd64_v0.12.0 -O subwasm.deb
+    wget https://github.com/chevdor/subwasm/releases/download/v0.12.0/subwasm_linux_amd64_v0.12.1 -O subwasm.deb
```
Comment (chevdor) [inline README.md:53]: You missed one here :)

**Review by chevdor — CHANGES_REQUESTED** [review]
Make sure all versions are bumped please

File: README.md (hunk 1 above)
Comment (sacha-l) [reply README.md]: Should be all good now.

**Comment by chevdor** [comment]
So for this PR, the md should not be fixed, instead, the .adoc should be updated and you run `just md` to get the markdown version. The `asciidoc` version is less error prone (and prevent the misses you experienced) as it uses a variable. As far as updating the usages (which are `includes` in the asciidoc source), they can be refreshed by running `just doc` or `just usage`. Not only it will bump the versions but also ensure that the doc is in sync.

I will be pushing a new commit to your PR soon.

**Pull request merged**